store/backup.go

```diff
@@ -395,9 +395,9 @@ func (s *Store) createBackupImpl(ctx context.Context, tx *sql.Tx, create *api.Ba
 			migration_history_version,
 			path
 		)
-		VALUES ($1, $2, $3, $4, 'PENDING_CREATE', $5, $6, $7, $8)
+		VALUES ($1, $2, $3, $4, '%s', $5, $6, $7, $8)
 		RETURNING id, creator_id, created_ts, updater_id, updated_ts, database_id, name, status, type, storage_backend, migration_history_version, path, comment
-	`,
+	`, api.BackupStatusPendingCreate),
```

Comment:
Shall we still pass it as SQL prepare statement argument? I know it's not vulnerable per SQL injection see, but just to be consistent with the rest of arguments.